### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by varnikaaavyas** [community/community#120709] [GitHub Education]
### Hi everyone,

We're still processing your application. Once this is complete you will have access to the student developer pack offers.
![image](https://github.com/community/community/assets/65498318/68740d64-2238-442e-9092-84e517d0c4f7)

Its showing me this, i applied 3 days back. Can anyone help me out?

**Comment by tr41z** [comment]
It may take up to couple weeks. Just wait until they send you confirmation email saying that everything is complete.

**Reply by samus-aran** [reply]
Exactly @traiz-dev 👍 Awesome answer (though hopefully not weeks) ⭐

**Comment by samus-aran** [comment]
Hi @varnikaaavyas 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Comment by kanduru-abhiram** [comment]
Hi @varnikaaavyas, please check whether your Application request is approved and the background color of Approved in green.

![image](https://github.com/community/community/assets/85011280/243cdc64-5fc7-4443-aeac-c36224d65c0b)

And wait till the background color of Approved turns to Violet, once turned, you'll get the access to all the benefits of Student Developer pack.

![image](https://github.com/community/community/assets/85011280/8809cebe-d954-4843-9e2f-4ef38cc5b082)

Please accept this as a solution if helped, thanks.
